fronts-client/src/lib/createAsyncResourceBundle/index.ts

```diff
@@ -246,7 +246,7 @@ function createAsyncResourceBundle<Resource>(
   const selectLastFetchOrder = (state: RootState): string[] =>
     selectLocalState(state).lastFetchOrder || defaultArray;
 
-  const selectAll = (state: RootState) => selectLocalState(state).data;
+  const selectAll = (state: RootState) => selectLocalState(state)?.data || {};
```

Comment:
Is this change still necessary? I can't remember whether we still need it, or whether it's an artefact of the journey we took to get here 😀